### PR TITLE
Support test case about create index

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

**Comment by Shylock-Hg** [comment]
The mock testing will not maintain again, please write cases in UT or Integrate Test(pytest).

File: src/context/QueryContext.h [hunk 1]
```diff
@@ -105,6 +105,10 @@ class QueryContext {
         return sm_;
     }
 
+    meta::IndexManager* indexMng() const {
```
Comment (yixinglu) [inline src/context/QueryContext.h:108]: Prefer `indexMgr`

File: src/validator/MaintainValidator.cpp [hunk 2]
```diff
@@ -319,13 +319,18 @@ Status CreateTagIndexValidator::validateImpl() {
...
 
+    // auto space = vctx_->whichSpace().id;
     auto status = Status::OK();
     do {
-        auto schema = vctx_->getSchema(name_);
-        if (schema == nullptr) {
-            status = Status::Error("Tag %s not exist", name_.c_str());
-            break;
-        }
+        // auto edgeStatus = qctx_->schemaMng()->toTagID(space, name_);
```
Comment (yixinglu) [inline src/validator/MaintainValidator.cpp:325]: Why to remove these implementation ?

File: src/mock/test/IndexTest.cpp [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        client_.reset();
+    }
+
+protected:
+    static std::unique_ptr<GraphClient>     client_;
+};
+
+std::unique_ptr<GraphClient> IndexTest::client_{nullptr};
+
+TEST_F(IndexTest, TagIndex) {
```
Comment (yixinglu) [inline src/mock/test/IndexTest.cpp:41]: Need to add similar test cases in our CI with pytest

File: src/context/QueryContext.h (hunk 1 above)
Comment (darionyaphet) [reply src/context/QueryContext.h:108]: Because `schemaMng` for schema manager

**Review by jievince — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**